server/ntb/io/feed_parsers/afp_newsml.py

```diff
@@ -40,5 +40,14 @@ def parse(self, xml, provider=None):
 
         return item
 
+    def parse_newslines(self, item, tree):
+        if super().parse_newslines(item, tree):
```

Comment:
Why do we need this condition and why we return True ?

Reply:
good question, the parent method returns true but it's not checked anywhere. will get rid of it